lib/puppet/util/log.rb

```diff
@@ -340,10 +340,6 @@ def to_structured_hash
     hash
   end
 
-  def to_pson(*args)
```

Comment:
Since Puppet::Util::Log includes Puppet::Network::FormatSupport, and the latter implements `to_pson` (in the same way), then I think this is a noop.